backend/tracim_backend/command/search.py

```diff
@@ -36,9 +36,11 @@ def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvir
         self.search_api.create_index()
         print("Index template was created")
         if parsed_args.index_all:
-            print("Indexing all content")
-            self.search_api.index_all_content()
-            print("All content where indexed")
+            nb_errors = self.search_api.index_all_content()
+            if nb_errors == 0:
+                print("All content where indexed")
```

Comment:
you should write the number of indexed contents.

Reply:
done.